osu.Game.Tests/Visual/UserInterface/TestSceneDeleteLocalScore.cs

```diff
@@ -169,7 +169,7 @@ public void TestDeleteViaRightClick()
             AddStep("click delete button", () =>
             {
                 InputManager.MoveMouseTo(dialogOverlay.ChildrenOfType<DialogButton>().First());
-                InputManager.Click(MouseButton.Left);
+                InputManager.PressButton(MouseButton.Left);
```

Comment:
test should release the button at the end as a means of cleaning up after itself - i believe you'll find that if you run this test twice in succession, it will fail as written on the second go

Reply:
Ah, I assumed that it would reset that, will do.